packages/nimble-blazor/NimbleBlazor.Acceptance.Tests/Pages/_Layout.cshtml

```diff
@@ -0,0 +1,32 @@
+﻿@using Microsoft.AspNetCore.Components.Web
+@namespace NimbleBlazor.Tests.Acceptance.Pages
+@addTagHelper *, Microsoft.AspNetCore.Mvc.TagHelpers
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <base href="~/" />
+    <link href="css/site.css" rel="stylesheet" />
```

Comment:
Some comments about styles:
1. I cloned your branch to my mac and changed the `npm run test -w @ni/nimble-blazor` command to run `Debug`. I noticed that the pages that load in Chromium seem to be missing Nimble fonts. According to the NimbleBlazor README you need to add: `<link href="_content/NimbleBlazor/nimble-tokens/css/fonts.css" rel="stylesheet" />`. (BTW, awesome this works on mac 👍)
2. can you take a pass at the existing CSS and remove anything that isn't necessary? For example in `site.css` there are some classes like `btn-primary` that don't apply to anything. And I'm not sure what `NimbleBlazor.Tests.Acceptance.styles.css` is but if it isn't needed we should delete it (it looks like other Blazor apps have equivalent code, so possibly I just don't understand Blazor's build output).

Reply:
Added the Nimble font CSS (and verified Source Sans Pro was loaded on one of the test Razor pages), pruned the site CSS (just kept the error styling).

`NimbleBlazor.Tests.Acceptance.styles.css` is auto generated as part of the Blazor build, and will contain the CSS from the Razor pages. So for this test project currently it'll contain the `.root` style from `MainLayout.razor.css`, modified a bit by Blazor CSS Isolation. So that one should stay.